pages/stylesheets/style.css

```diff
@@ -0,0 +1,158 @@
+.bg-red {
+    background-color: red;
+}
+
+.bg-blue {
+    background-color: blue;
+}
+
+
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    margin:20px;
+}
+
+
+.row {
+    margin-left: auto;
+    margin-right: auto;
+    padding-left: 15px;
+    padding-right: 15px;
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-gap: 0px;
+}
+
+.container {
+    width: 100vw;
+    margin-bottom: 50px;    
+}
+
+.col-1 {
+    grid-column: span 1;
+    padding: 5px;
+}
+.col-2 {
+    grid-column: span 2;
+    padding: 5px;    
+}
+.col-3 {
+    grid-column: span 3;
+    padding: 5px;
+}
+.col-4 {
+    grid-column: span 4;
+    padding: 5px;
+}
+.col-5 {
+    grid-column: span 5;
+    padding: 5px;
+}
+.col-6 {
+    grid-column: span 6;
+    padding: 5px;
+}
+.col-7 {
+    grid-column: span 7;
+    padding: 5px;
+}
+.col-8 {
+    grid-column: span 8;
+    padding: 5px;
+}
+.col-9 {
+    grid-column: span 9;
+    padding: 5px;
+}
+.col-10 {
+    grid-column: span 10;
+    padding: 5px;
+}
+.col-11 {
+    grid-column: span 11;
+    padding: 5px;
+}
+.col-12 {
+    grid-column: span 12;
+    padding: 5px;
+}
```

Comment:
Did you take these styles from some external source?  A 12-column grid is a pretty common design pattern.

Reply:
yes totally. I looked at bootstrap because it was mentioned in the lesson and found a page that describe grid columns and also talked to a friend who mentioned that newspapers were designed with 12 columns because they are divisible into 2,3, and 4; then they gave me a whole lesson about imperial measures being divisible this way for simplicity vs. metric...and that was enough for me.  :-)